src/pages/Login/container.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+
+import Login from './component';
+import { sessionSetId } from '../../../state/session/actions';
+
+const mapDispatchToProps = (dispatch) => ({
+  onSessionSetId: (id) => dispatch(sessionSetId(id))
```

Comment:
Also, call this as action - setSessionId

Reply:
Done.